### ISSUE TITLE
[Font Request] Wremena

### ISSUE THREAD

**Issue by blows** [KyleAMathews/typefaces#136]
It would be great to include [Wremena](http://typefaces.temporarystate.net/preview/Wremena) among the supported typefaces.

If there's a guide for how to contribute let me know and I'll have a go!
Thank you :)

**Comment by KyleAMathews** [comment]
Is it on fontsquirrel?

**Comment by blows** [comment]
Doesn't come up in the Font Squirrel search, so I guess not. Possibly just available from their free download page: http://typefaces.temporarystate.net/download

Is that a problem for including them in this project?

**Comment by KyleAMathews** [comment]
Yeah, it's more complicated to add — you'd need to PR a custom script to the scripts folder for this font.
